tutorials/haskell/golden-testing-in-haskell-with-hspec-gold/code/src/package.yaml

```diff
@@ -0,0 +1,54 @@
+name:                hspec-golden-tutorial
+version:             0.1.0.0
+github:              "githubuser/hspec-golden-tutorial"
+license:             BSD3
+author:              "Adrian Vera Medina"
+maintainer:          "adrian-860@hotmail.com"
+copyright:           "2019 Adrian Vera Medina"
+
+extra-source-files:
+- README.md
+- ChangeLog.md
+
+# Metadata used when publishing your package
+# synopsis:            Short description of your package
+# category:            Web
+
+# To avoid duplicated efforts in documentation and dealing with the
+# complications of embedding Haddock markup inside cabal files, it is
+# common to point users to the README.md file.
+description:         Please see the README on GitHub at <https://github.com/MedAdrian/hspec-golden-tutorial#readme>
```

Comment:
@MedAdrian Could you please update the url for the README?

Reply:
Should it the URL be the one inside the project?

Reply:
Yeah, I think that should work.